node/external/logs/logsRepository.go

```diff
@@ -17,6 +19,10 @@ type logsRepository struct {
 
 func newLogsRepository(storageService dataRetriever.StorageService, marshaller marshal.Marshalizer) *logsRepository {
 	storer := storageService.GetStorer(dataRetriever.TxLogsUnit)
+	if check.IfNil(storer) {
```

Comment:
It's a bit debatable if we should have used a `disabledLogsFacade` at a higher level, instead, when logs saving is disabled. But it's all right this way, as well :pray:

Reply:
pushed Iulian's solution which fixes this